bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/carousel/Carousel.java

```diff
@@ -147,7 +147,7 @@ protected void populateItem(ListItem<ICarouselImage> item) {
     protected Component newImage(final String markupId, final ICarouselImage image) {
         final Label img = new Label(markupId);
         img.add(new AttributeModifier("src", image.url()));
-
+        img.add(new AttributeModifier("class", "d-block w-100"));
```

Comment:
Wouldn't it be better to put this directly at https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/77255505c7c67e1c0fbebe0c1c98efa483617aca/bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/carousel/Carousel.html#L9 ?

Reply:
done